chain/network/src/network_protocol/network.proto

```diff
@@ -299,6 +299,8 @@ message RoutedMessage {
   bytes borsh = 1;
   // Timestamp of creating the Routed message by its original author.
   google.protobuf.Timestamp created_at = 2;
+  // Number of peers this routed message travelled through. Doesn't include the peer that created the message.
+  optional int32 num_hops = 3;
```

Comment:
in proto3 syntax all fields are "optional". In particular a missing value for field of integer type defaults to 0.

Reply:
i.e. please remove "optional"

Reply:
how can I tell apart the case of this value initialized to `0` and this field simply missing?
An alternative would be to start counting hops from `1`, but that introduces a magic value, which I don't like.

Reply:
ok, so yeah, there is a difference between optional and non-optional in proto3: https://developers.google.com/protocol-buffers/docs/proto3#specifying_field_rules . The usage is just discouraged, because it is too easy to confuse "not set" with "set to default value". You can leave it as it is now then.

Reply:
I see, thank you.